DASCharter-2020.html

```diff
@@ -1,677 +1,1357 @@
+            <tr>
+              <th>
+                Ambient Light Sensor
+              </th>
+              <td>
+                <a href=
+                "http://www.w3.org/TR/2012/WD-ambient-light-20120802/">2 Aug
+                2012</a>
+              </td>
+              <td>
+                <a href=
+                "https://www.w3.org/TR/2018/CR-ambient-light-20180320/">20 Mar
+                2018</a>
+              </td>
+              <td>
+                Q4 2021
+              </td>
+              <td>
+                Q1 2022
+              </td>
+            </tr>
+            <tr>
+              <th>
+                Accelerometer
+              </th>
+              <td>
+                <a href=
+                "https://www.w3.org/TR/2016/WD-accelerometer-20160913/">13
+                September 2016</a>
+              </td>
+              <td>
+                <a href=
+                "https://www.w3.org/TR/2019/CR-accelerometer-20191212/">12 Dec
+                2019</a>
+              </td>
+              <td>
+                Q4 2021
+              </td>
+              <td>
+                Q1 2022
+              </td>
+            </tr>
+            <tr>
+              <th>
+                Gyroscope
+              </th>
+              <td>
+                <a href="https://www.w3.org/TR/2016/WD-gyroscope-20160913/">13
+                September 2016</a>
+              </td>
+              <td>
+                <a href="https://www.w3.org/TR/2019/CR-gyroscope-20191212/">12
+                Dec 2019</a>
+              </td>
+              <td>
+                Q4 2021
+              </td>
+              <td>
+                Q1 2022
+              </td>
+            </tr>
+            <tr>
+              <th>
+                Magnetometer
+              </th>
+              <td>
+                <a href=
+                "https://www.w3.org/TR/2016/WD-magnetometer-20160913/">13
+                September 2016</a>
+              </td>
+              <td>
+                <a href=
+                "https://www.w3.org/TR/2018/CR-magnetometer-20180320/">20 Mar
+                2018</a>
+              </td>
+              <td>
+                Q4 2021
+              </td>
+              <td>
+                Q1 2022
+              </td>
+            </tr>
+            <tr>
+              <th>
+                Orientation Sensor
+              </th>
+              <td>
+                <a href=
+                "https://www.w3.org/TR/2017/WD-orientation-sensor-20170511/">11
+                May 2017</a>
+              </td>
+              <td>
+                <a href=
+                "https://www.w3.org/TR/2019/CR-orientation-sensor-20191212/">12
+                Dec 2019</a>
+              </td>
+              <td>
+                Q4 2021
+              </td>
+              <td>
+                Q1 2022
+              </td>
+            </tr>
+            <tr>
+              <th>
+                Fold Angle Sensor
```

Comment:
Should this pass through incubation before being included here? or included here in anticipation that it will go through incubation during the course of our chartered period?

Reply:
The latter.